template/tasks.py

```diff
@@ -224,6 +224,13 @@ def run_docs(ctx):
     ctx.run("poetry run mkdocs serve")
 
 
+@invoke.task
+def install(ctx, install_playwright: bool = False):
+    ctx.run("pip install poetry")
+    ctx.run("poetry install")
+    if install_playwright:
+        ctx.run("poetry run playwright install --with-deps")
```

Comment:
<!-- Issues highlight specific problems with the subject under review. These problems can be user-facing or behind the scenes. It is strongly recommended to pair this comment with a suggestion. If you are not sure if a problem exists or not, consider leaving a question.  -->
**issue:** 

Remember that these tasks server 2 purposes - for calling during CI jobs but also for helping the developer locally with general day to day tasks. I feel like this task has been written with only the CI aspect considered.
‏
I think we can assume that poetry itself is installed and the generally preference for local development is to install it with the poetry installed _not_ pip. That way it gets it's own virtual environment independent from the individual project that is manages. So we should not pip install it here.

I can see what you are trying to do with the playwright install flag but I think it might be better the opposite way around. For developers locally I think that the expectation they would have is that you run this command and it install the application and all it's dependencies. That would include playwright. I can see that in CI you probable only want to install playwright for the functional tests. So perhaps flip the parameter around so it installs by default but can be turned _off_ when we are certain we will not need it?

Reply:
I agree with removing the pip command. 👍🏻 

Re the default behaviour of the task to install dependancies, I was hesitated which way to go. Your point of view make sense to me. I'll update the flag to something like `--skip-install-playwright`.

Reply:
Updated in [28db98c](https://github.com/ackama/django-template/pull/46/commits/28db98cc5520755da41e5662dab813a876d33d2c)